lib/utils.js

```diff
@@ -132,4 +133,23 @@ async function safeRimRaf (delPath, tryNum = 0) {
   }
 }
 
-export { killAllSimulators, endAllSimulatorDaemons, safeRimRaf, simExists };
+async function installSSLCert (pemText, udid) {
+  let tempFileName = `${__dirname}/temp-ssl-cert.pem`;
+  // TODO: Confirm if this is universally applicable to all versions of OSX
+  let pathToKeychain = `${process.env.HOME}/Library/Developer/CoreSimulator/Devices/${udid}/data/`;
```

Comment:
pretty sure you don't need to construct this dir here, I think we have methods already which get you a path like this from the udid?